lang/java/avro/src/test/java/org/apache/avro/TestFixed.java

```diff
@@ -0,0 +1,19 @@
+package org.apache.avro;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TestFixed {
+
+
+  @Test
+  public void testFixedDefaultValueDrop() {
```

Comment:
The code looks good to me.
My concerns are with the tests. 
Why is this a new class and not merged into SchemaBuilder?